src/parser_utils/type_sig/dune

```diff
@@ -0,0 +1,17 @@
+(library
+ (name flow_type_sig)
+ (wrapped false)
+ (libraries
+  flow_parser_utils
+  cycle_hash
+  dtoa
+  flow_common
+  flow_common_utils
+  flow_common_utils_loc_utils
+  flow_common_tarjan
```

Comment:
I'm not looking closely at which dependencies you are adding, but I'm pretty familiar with this code and I'm surprised to see the dependency on tarjan here. I don't think we depend on this library.

Reply:
indeed, flow_common_tarjan is not needed, fixed
There is an issue upstream https://github.com/ocaml/dune/issues/650, once the feature request is landed, we can find it automatically